src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -53,6 +53,23 @@ public void MakeCompilationOutputRunnable()
             ExportRuntimeAssets();
         }
 
+        private void VerifyCoreclrPresenceInPackageGraph()
```

Comment:
Looking at the Constants.cs file, it already uses CoreClr.

My opinion is since CLR is an acronym, it is separate from "core", thus in PascalCasing would require a new captial letter.

---
In reply to: [61924689](https://github.com/dotnet/cli/pull/2781#discussion_r61924689) [](ancestors = 61924689)